wire-ios-sync-engine/Tests/Source/Repositories/ConnectionsRepositoryTests.swift

```diff
@@ -0,0 +1,154 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+@testable import WireAPI
+import WireAPISupport
+import WireDataModelSupport
+@testable import WireSyncEngine
+import WireSyncEngineSupport
+import XCTest
```

Comment:
Does it make sense to sort `@testable` in their own block to make it easier to read?
```suggestion
import WireAPISupport
import WireDataModelSupport
import WireSyncEngineSupport
import XCTest

@testable import WireAPI
@testable import WireSyncEngine
```